WSLDiskShrinker/Common/BooleanToInverseVisibilityConverter.cs

```diff
@@ -5,15 +5,13 @@ namespace WSLDiskShrinker.Common;
 
 class BooleanToInverseVisibilityConverter : IValueConverter
 {
-	public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
-	{
-		bool v = (bool) value;
-		return v ? Visibility.Collapsed : Visibility.Visible;
-	}
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return (bool)value ? Visibility.Collapsed : Visibility.Visible;
+    }
 
-	public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
-	{
-		throw new NotImplementedException();
-	}
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return (bool)value ? Visibility.Visible : Visibility.Collapsed; // Not used.
+    }
```

Comment:
This implementation of `ConvertBack` is not correct. What `ConvertBack` should do is converting from `Visibility` to `bool`, which is useless for our case. (FYI: It is useful in two-way bindings)